datarobot_batch_scoring/batch_scoring.py

```diff
@@ -121,10 +122,17 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
         base_headers['content-type'] = 'text/csv; charset=utf8'
         if compression:
             base_headers['Content-Encoding'] = 'gzip'
+
         if import_id:
-            endpoint = base_url + '/'.join((import_id, 'predict'))
+            endpoint = base_url + '/import_id'
```

Comment:
this one does not look right, you have `import_id` literal instead of value

Reply:
You are right. Copy/pasta failure.